<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>380</width>
         <height>545</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Menu Atual: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="menusCb"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="reclassifyCkb">
          <property name="text">
           <string>reclassificar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchButtonLe">
          <property name="placeholderText">
           <string>Pesquisar Bot√£o</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="menuArea" native="true">
          <layout class="QVBoxLayout" name="menuLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
